### DISCUSSION TITLE
HOWTO : install on Ubuntu 24.04 with PHP 8.3 and MariaDB 10.6

### DISCUSSION THREAD

**Discussion by Giuliano69** [opensourcepos/opensourcepos#4054] [Q&A]
Hi,
is there a version (under development) of OpensourcePOS  that could be installed with:
Ubuntu 24.04 +  **PHP 8.3 +  MariaDB 10.6** ?  (These versions are the ones that comes with last 24.04 LTS ubuntu version ...)

The  INSTALL.md file, says that 

- _"PHP version 8.x is only supported by the code that is still under **development** in master branch."_  .
-  _"also MariaDB replacement 10.x is supported and might offer better performance."_

But I can NOT  find a "development"  version to install under branches/tags menu..

KINDLY ASK:
Which version should I choose from branches/tags menu to have it work with PHP8.3 and MariaDB 10.6 ?

Thanks in advance for any help.

Giuliano

**Comment by jekkos** [comment]
Hi there used to be unstable builds pushed to npm but they are not updated anymore. You can clone the repo and try to build it from source. I will check if I can enable this upload again.

**Comment by Giuliano69** [comment]
Hi jekkos
thanks for your reply.

Do you mean that the "master" branch could work with PHP 8.3 amd Mariadb 10.6 ?

**Reply by jekkos** [reply]
Correct there might be still some bugs, feel free to report back here.
